### ISSUE TITLE
Add PSP_SYSTEM_VER to virtual PBP implementation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kuuhakuzx/procfw#281]
```
What is the expected output?
Minimun firmware version shown in xmb plugins who read games.

What do you see instead?
Shows 1.00 by default.

What version of the product are you using? On what operating system?
6.60 Pro B9

Please provide any additional information below.
The attached patch adds the correct PSP_SYSTEM_VER to the virtual pbp 
implementation as seen in other CFW/HEN who implements this feature.
```

Original issue reported on code.google.com by `codestat...@gmail.com` on 11 Sep 2011 at 1:39

Attachments:
- [add_psp_system_ver_virtual_pbp.patch](https://storage.googleapis.com/google-code-attachments/procfw/issue-281/comment-0/add_psp_system_ver_virtual_pbp.patch)

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the patch.
```

Original comment by `devnonam...@gmail.com` on 15 Jun 2012 at 4:02
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
I dont understand how this patch is ment to change anything...

If you look into vshmain.prx disassembly you will see that the XMB auto-sets 
the required firmware version to 1.0 if the tag isn't found... which is also 
the reason why PROCFW doesn't fake the tag.

This issue will remain open for another week or so... to give the OP a chance 
to reason why this patch should be accepted non the less.

If no response is given within a week I will close this issue.
```

Original comment by `gentlema...@gmail.com` on 16 Jun 2012 at 10:30

**Comment by GoogleCodeExporter** [comment]
```
The patch is only a cosmetic fix for plugins that hook the virtual pbp funcs to 
read the iso atrributes. The only goal of the patch was to get consistent info 
from the virtual pbp between different implementations of CFW (for example the 
ME CFW provides PSP_SYSTEM_VER for virtual pbp).

Of course the "plugins" list that i mentioned is reduced to one that i ported 
(i haven't followed the psp scene lately so i dunno if other plugins that read 
need to read PSP_SYSTEM_VER). The PSP_SYSTEM_VER is purely informative and 
serves no purpose in current CFW implementations other that show the base FW 
that a game needs to run.

At the time of writting the patch i was the only affected and the only one 
reading the value so feel free to implement the patch or reject it again :), 
the patch isn't gonna dissapear and anybody who need this can patch the source 
since is almost a one line patch (highly unlikely that is gonna get out of sync 
with the current source tree).

tl;dr: the patch is a purely cosmetic fix for plugin makers that for whatever 
reason needs to hook the virtual pbp funcs to get the info.

```

Original comment by `codestat...@gmail.com` on 17 Jun 2012 at 12:41

**Comment by GoogleCodeExporter** [comment]
```
I think it's a good idea to add this patch because it would allow users to see 
which version is supposed to be required.
Knowing that the Pro Online client doesn't work the same for all FWs (version 
0.09 is advised for > 1.00 games whereas version 0.10 is needed for 1.00 
games), it would allow users to better recognize the firmware originally 
required by the game (even though this information is not 100% reliable)
```

Original comment by `devnonam...@gmail.com` on 26 Aug 2013 at 12:06
- Added labels: **Priority-Low**
- Removed labels: **Priority-Medium**
